core/templates/dev/head/components/question-directives/question-player/question-player.directive.html

```diff
@@ -17,13 +17,47 @@ <h3 class="oppia-question-player-header">Question <[$ctrl.currentQuestion]>/<[$c
   <!-- Results Page -->
   <div class="oppia-question-player-results" ng-if="resultsLoaded">
     <div class="oppia-question-player-card question-player-results-header">
-      <h3 class="oppia-question-player-header"> Test Complete. Well Done! </h3>
-      <score-ring score="$ctrl.totalScore"></score-ring>
-      <h4 class="learn-more"> Learn more about your score </h4>
+      <h3 class="oppia-question-player-header" ng-if="$ctrl.isTestPassed"> Test Complete. Well Done! </h3>
```

Comment:
Names that start with "is" usually refer to a function name. Please follow this for booleans.

```suggestion
      <h3 class="oppia-question-player-header" ng-if="$ctrl.testIsPassed"> Test Complete. Well Done! </h3>
```

Reply:
Done.